kitchensink/index.html

```diff
@@ -1800,9 +1800,21 @@ <h3>Required Fields</h3>
                     <div class="mdl-textfield mdl-js-textfield mdl-textfield--full-width is-required is-readonly">
                       <input class="mdl-textfield__input" type="text" pattern="[A-Z,a-z]*" id="sample4999" aria-required="true" readonly/>
                       <label class="mdl-textfield__label static-label" for="sample4999">static-label readonly<span class="required-field"></span></label>
-                      <span class="mdl-textfield__error"><i class="fa fa-exclamation-triangle fa-fw"></i>&nbsp;Only alphabet and no spaces</span>
+              <span class="mdl-textfield__error"><i class="fa fa-exclamation-triangle fa-fw"></i>&nbsp;Only alphabet and no spaces</span>
                     </div>
                   </form>
+                  
+                  <!-- Form Input Type File element -->
```

Comment:
In general, it would be good to include a pic/gif of the change in the PR.

Reply:
gotcha, a screenshot of the updates has been added to the conversation.